app-management/src/main/java/cf/dmms/appmanagement/tools/FilesArchiver.java

```diff
@@ -0,0 +1,40 @@
+package cf.dmms.appmanagement.tools;
+
+import java.io.*;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class FilesArchiver {
+
+	public static File zipFiles(List<File> files) throws IOException {
+		File output = File.createTempFile("DMMS-logs", ".zip");
+		ZipOutputStream zipOut = zipOutputStream(output);
```

Comment:
IMO it would be better to use try with resources here (abandoning this method call will be inevitable unfortunately).